src/test/java/de/mediathekview/mserver/crawler/CrawlerManagerFTPTest.java

```diff
@@ -42,10 +42,8 @@
   private static final String FILMLISTS_TARGET_BASE_PATH = "/var/www/mediathekview/filmlisten/";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Refactor your code to get this URI from a customizable parameter. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1075)